cypress/support/index.ts

```diff
@@ -22,6 +22,8 @@ declare global {
       dataCy(value: string): Chainable<Element>;
 
       stubMoter(state: MoteState): Chainable<Element>;
+
+      OAuth2Login(): Chainable<Element>;
```

Comment:
Man trenger egentlig ikke å legge den til her, Cypress-testene kjører fint uansett. Men "Build"-steget i Actions feiler fordi  `TS2339: Property 'OAuth2Login' does not exist on type 'cy & EventEmitter'.`.